pkg/quota/admission/clusterresourceoverride/admission.go

```diff
@@ -180,3 +176,39 @@ func (a *clusterResourceOverridePlugin) Admit(attr admission.Attributes) error {
 func multiply(x *inf.Dec, y *inf.Dec) *inf.Dec {
 	return inf.NewDec(0, 0).Mul(x, y)
 }
+
+// wrappedLimitRangerActions wraps the default LimitRangerActions.
+type wrappedLimitRangerActions struct {
+	internalLimitRangerActions limitranger.LimitRangerActions
+}
+
+// wrappedLimitRangerActions implements the LimitRangerActions interface
+var _ limitranger.LimitRangerActions = &wrappedLimitRangerActions{}
+
+// Limit is a pluggable function to enforce limits on the object.  This wraps the default
+// implementation to always return success so that all defaults will be applied.  Validation will
+// occur after the overrides.  Implements the LimitRangerActions interface.
+func (lr *wrappedLimitRangerActions) Limit(limitRange *kapi.LimitRange, kind string, obj runtime.Object) error {
+	lr.internalLimitRangerActions.Limit(limitRange, kind, obj)
+	return nil
+}
+
+// SupportsLimit provides a check to see if the limitRange is applicable.
+// Implements the LimitRangerActions interface.
+func (lr *wrappedLimitRangerActions) SupportsLimit(limitRange *kapi.LimitRange) bool {
+	return lr.internalLimitRangerActions.SupportsLimit(limitRange)
+}
+
+// SupportsAttributes is a helper that returns true if the resource is supported by the plugin.
+// Implements the LimitRangerActions interface.
+func (lr *wrappedLimitRangerActions) SupportsAttributes(attr admission.Attributes) bool {
+	return internalSupportsAttributes(attr)
+}
+
+// internalSupportsAttributes is factored out for usage outside the actions object.
+func internalSupportsAttributes(attr admission.Attributes) bool {
+	if attr.GetResource() != kapi.Resource("pods") || attr.GetSubresource() != "" {
```

Comment:
nit: would be better factored as `return <boolean condition>`.  Harder to read this way.